site/docs/0.0/components/navbar.md

```diff
@@ -39,7 +39,6 @@ Here's an example of all the sub-components included in a responsive light-theme
 ## Example
 
 Include example of "default" navbar (`.navbar-light`).
-The use of a `button` element is recommended for the `.dropdown-toggle` in your navbar, given that your dropdown will open on click.
```

Comment:
I think it might be a mistake that all of these changes were reverted to this file. We decided to add in these instructions for `button`s (and change the corresponding examples) for the sake of semantics and accessibility; to not have `a` elements as the dropdown parent, because our dropdowns activate on click and not on hover.